bots/elasticsearch/tasks.py

```diff
@@ -18,6 +21,22 @@ def safe_substr(value, length=32000):
         return str(value)[:length]
     return None
 
+def add_suggest(obj):
+    input = obj['name']
+    if input:
+        # Elastic's completion suggester only looks at prefixes. If we give each
```

Comment:
Ah, you read closer than I do. So this will turn `'Foo Bar Baz'` into `'['Bar Baz', 'Baz']`?
Would it be better to split on whitespace? `'Foo Bar Baz'` -> `['Foo', 'Bar', 'Baz']`